lib/Controller/SignalingController.php

```diff
@@ -216,6 +216,11 @@ public function getWelcomeMessage(int $serverId): DataResponse {
 				],
 			]);
 
+			$featureHeader = $response->getHeader('X-Spreed-Signaling-Features');
+			if (strpos($featureHeader, 'audio-video-permissions') === false) {
```

Comment:
While this works for now, better would be to split the comma-separated string and check the individual items.

Reply:
it's `, ` (comma + space) seperated, right?
I will split by command then then trim to be save